SmartDeviceLinkTests/RPCSpecs/StructSpecs/SDLDisplayCapabilitiesSpec.m

```diff
@@ -52,7 +52,7 @@
     });
     
     it(@"Should get correctly when initialized", ^ {
-        NSMutableDictionary* dict = @{SDLRPCParameterNameDisplayType:SDLDisplayTypeGen26DMA,
+        NSDictionary* dict = [@{SDLRPCParameterNameDisplayType:SDLDisplayTypeGen26DMA,
```

Comment:
If you change this to an NSDictionary, you can't / shouldn't `mutableCopy`. Remove all `mutableCopy`s from this dictionary.